HHS/OPRE-OPS
# style: 💅 allow for neg values in Budget Card

## Issue

#2842 

## How to test

1. update CANSummaryCards totalSpending to over total budget
2. see neg value

## Screenshots

<img width="499" alt="image" src="https://github.com/user-attachments/assets/0de3e692-8c23-4b36-be05-3e3700a14a52">

## Definition of Done Checklist
